src/Application.php

```diff
@@ -58,6 +58,9 @@ class Application extends SymfonyApplication implements LoggerAwareInterface, Co
     /** @var TildeExpansionHook */
     protected $tildeExpansionHook;
 
+    /** @var string[] */
+    protected $bootstrapCommandClasses = [];
```

Comment:
Lets please use an actual type hint as well. 
```suggestion
    protected array $bootstrapCommandClasses = [];
```